src/GovUk.Education.ExploreEducationStatistics.Content.Model/File.cs

```diff
@@ -22,6 +22,8 @@ public class File : ICreatedTimestamp<DateTime?>
 
         public int? DataSetFileVersion { get; set; }
 
+        public Guid? PublicDataSetVersionId { get; set; }
```

Comment:
I think it would be a good idea to add an index to this field by changing ContentDbContext with something like `entity.HasIndex(e => e.PublicDataSetVersionId)`;

Reply:
Yes, it 100% would be! I intended on doing this and forgot.

Added it now :)